android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java

```diff
@@ -184,10 +184,8 @@ public LinearLayout createCard(int type, int position) {
         LinearLayout layout1 = new LinearLayout(mActivity);
         LinearLayout layout2 = new LinearLayout(mActivity);
         LinearLayout layout3 = new LinearLayout(mActivity);
-
-        LinearLayout.LayoutParams linearLayoutParams = new LinearLayout.LayoutParams(
-                LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.WRAP_CONTENT);
-        linearLayoutParams.setMargins(mHorizontalMargin, 0, mHorizontalMargin, 7 * MARGIN_VERTICAL);
+        RecyclerView.LayoutParams linearLayoutParams =
```

Comment:
i think this would resolve the crash with scroll event

Reply:
@deeppandya yeah that was part of the issue